python/ml4ir/applications/ranking/model/ranking_model.py

```diff
@@ -251,7 +280,7 @@ def evaluate(
             df_group_metrics_summary = df_group_metrics.describe()
             self.logger.info(
                 "Computing group metrics using keys: {}".format(
-                    self.feature_config.get_group_metrics_keys("node_name")
+                    eval_dict["group_by"]
```

Comment:
@mohazahran Why change this?

Reply:
because in the eval yaml you can choose the group keys, so I'm using them rather than this. If none is provided in the yaml file, then `self.feature_config.get_group_metrics_keys("node_name")` will be used.